src/shared/varlink-io.systemd.service.c

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include "varlink-io.systemd.service.h"
+
+static VARLINK_DEFINE_METHOD(Ping);
+
+static VARLINK_DEFINE_METHOD(Reload);
+
+static VARLINK_DEFINE_METHOD(
+                SetLogLevel,
+                VARLINK_DEFINE_INPUT(level, VARLINK_INT, VARLINK_NULLABLE));
+
+VARLINK_DEFINE_INTERFACE(
+                io_systemd_service,
```

Comment:
Shouldn't this be hooked up to a varlink server somewhere via `varlink_server_add_interface()`?

Reply:
It shouldn't, because nothing implements the interface yet.